docker/start-app.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# This configuration is a temporary workaround until we migrate to CCF 4.0.3, which supports
+# a cchost "config-timeout" option to accomplish a similar result. This logic is required when
+# the SCITT container is deployed in Kubernetes without any orchestration sidecars.
+
+WAIT_TIME_SEC=3
+TIMEOUT_SEC=300
+
+echo "Starting app"
+config_arg=$1
+
+cchost --check ${config_arg}
+code=$?
+
+start_time=$(date +%s)
+
+while [[ $code -ne 0 ]]; do
+    
+    # Exit if timeout is reached
+    if [[ $(($(date +%s) - $start_time)) -ge $TIMEOUT_SEC ]]; then
```

Comment:
simplify and move out `$(date +%s)` into a local variable, there is also an issue with the use of `$` in `$(())` documented in https://www.shellcheck.net/wiki/SC2004

Reply:
Done, forgot to install the shellcheck extension in the GH codespace to catch these type of issues! 😄